go.mod

```diff
@@ -4,21 +4,17 @@ go 1.13
 
 require (
 	github.com/container-storage-interface/spec v1.0.0
-	github.com/davecgh/go-spew v1.1.1 // indirect
```

Comment:
I am not sure why there are so many changes in the `go.mod` and `go.sum` files. I executed `go mod tidy`, probably coming from that command.